crates/stc_ts_file_analyzer/tests/pass/types/conditional/conditionalTypes1/3.swc-stderr

```diff
@@ -6,7 +6,8 @@
    `----
 
 Error: 
-  > T extends (null | undefined) ? never : T
+  > (T & {
+  | })
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-8.html

Reply:
Yeah I know the reason